packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -391,7 +391,9 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
 
       Widget expandIconPadded = Padding(
         padding: const EdgeInsetsDirectional.only(end: 8.0),
-        child: ExpandIcon(
+        child: IgnorePointer(
+          ignoring: child.canTapOnHeader,
+          child: ExpandIcon(
           color: widget.expandIconColor,
           disabledColor: child.canTapOnHeader ? widget.expandIconColor : null,
           isExpanded: _isChildExpanded(index),
```

Comment:
Lines 397-405 should be indented, now that the `ExpandIcon` is wrapped with the `IgnorePointer` widget.